Keep awscrt pinned. This is the only user, and presumably
Amazon has a good reason for not bumping the version yet.

diff -ru aws-cli-2.11.0-orig/pyproject.toml aws-cli-2.11.0/pyproject.toml
--- aws-cli-2.11.0-orig/pyproject.toml	2023-03-02 19:00:36.000000000 -0500
+++ aws-cli-2.11.0/pyproject.toml	2023-03-03 11:05:13.214881849 -0500
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-"flit_core>=3.7.1,<3.8.1",
+"flit_core",
 ]
 build-backend = "pep517"
 backend-path = ["backends"]
@@ -28,21 +28,21 @@
     "Programming Language :: Python :: 3.11",
 ]
 dependencies = [
-    "colorama>=0.2.5,<0.4.7",
-    "docutils>=0.10,<0.20",
-    "cryptography>=3.3.2,<39.0.3",
-    "ruamel.yaml>=0.15.0,<=0.17.21",
+    "colorama",
+    "docutils",
+    "cryptography",
+    "ruamel.yaml",
     # ruamel.yaml only requires ruamel.yaml.clib for Python versions
     # less than or equal to Python 3.10. In order to ensure we have
     # a consistent dependency closure across all Python versions,
     # we explicitly include ruamel.yaml.clib as a dependency.
-    "ruamel.yaml.clib>=0.2.0,<=0.2.7",
-    "prompt-toolkit>=3.0.24,<3.0.29",
-    "distro>=1.5.0,<1.6.0",
-    "awscrt>=0.12.4,<0.17.0",
-    "python-dateutil>=2.1,<3.0.0",
-    "jmespath>=0.7.1,<1.1.0",
-    "urllib3>=1.25.4,<1.27",
+    "ruamel.yaml.clib",
+    "prompt-toolkit",
+    "distro",
+    "awscrt",
+    "python-dateutil",
+    "jmespath",
+    "urllib3",
 ]
 dynamic = ["version"]

